tensorflow_serving/model_servers/main.cc

```diff
@@ -265,6 +272,42 @@ class PredictionServiceImpl final : public PredictionService::Service {
   bool use_saved_model_;
 };
 
+class ModelManagement final : public ModelManagementService::Service {
```

Comment:
Could this class be merged with ModelManagementImpl? It seems there is an unnecessary level of indirection.

Reply:
I really don't know... If I remove separate service class, main.cc would be more complex, if move everything to separate service class I won't have access to DeadlineToTimeoutMillis method.

Reply:
Leave as-is then.